spec/lib/swagger_v2_api_declaration.json

```diff
@@ -142,16 +152,6 @@
           "text/xml",
           "text/html"
         ],
-        "parameters": [
```

Comment:
Sounds fine to me! As long as parameters in operations are still exercised somewhere I'm happy.